src/Services/MathService.ts

```diff
@@ -0,0 +1,32 @@
+import express, { Request, Response,NextFunction} from 'express'
+export const Handler = ( req: Request, response: Response,next: NextFunction) => {
+    const operation = req.query.op;
+    const a = req.query.n1;
+    const b = req.query.n2;
+    switch (operation) {
+        case 'ADD' :
+            const sum =Number(a) +Number(b)
+      response.status(200).send(`la somme de ${a} + ${b} egale :${sum} `)
+            break;
+
+        case 'MULT':
+            const sum2 =Number(a) *  Number(b)
+            response.status(200).send(`la multiplication  de ${a} * ${b} egale :${sum2} `)
+            break;
+
+         case 'DIV' :
+         const sum3=Number(a) +Number(b)
```

Comment:
Please check the operation you're done here, I think you should do div(/) here not plus(+) 
and always consider defining the variables with more significant wording